docs/spelling_wordlist.txt

```diff
@@ -233,3 +239,56 @@ yaml
 yml
 Zenodo
 連句
+cls
```

Comment:
Probably should sort this file alphabetically.

Reply:
I left these at the bottom on purpose so i know where to begin for fixing them in renku-python

Reply:
ok I went ahead and sorted it anyway in case something unpredictable happens with this follow-up PR :)